frontend/src/features/admin-form/template/DuplicateFormTemplateModal/DupeFormTemplateWizardProvider.tsx

```diff
@@ -0,0 +1,126 @@
+import { useEffect, useMemo } from 'react'
+
+import { FormResponseMode } from '~shared/types'
+
+import { useFormTemplate } from '~features/admin-form/common/queries'
+import { isMyInfo } from '~features/myinfo/utils'
+import {
+  CreateFormFlowStates,
+  CreateFormWizardContext,
+  CreateFormWizardContextReturn,
+} from '~features/workspace/components/CreateFormModal/CreateFormWizardContext'
+import { useCommonFormWizardProvider } from '~features/workspace/components/CreateFormModal/CreateFormWizardProvider'
+
+import { useDuplicateFormTemplateMutations } from '../mutation'
+
+export const useDupeFormTemplateWizardContext = (
```

Comment:
similarly we should keep the terminology ("duplicate form template" vs "use template") consistent. suggest `useUseTemplateWizardContext`. `useUse` is weird but at least avoids confusion with "duplicate form" which is a different action